### DISCUSSION TITLE
litellm.exceptions.APIConnectionError: {"detail":"Not Found"} when trying to call completion

### DISCUSSION THREAD

**Discussion by techybolek** [ModelTC/LightLLM#240] [General]
I have installed ollama and downloaded mistral. I've launched ollama with mistral and then started light llm:

litellm --model ollama/mistral

```
I've verified with curl that it runs correctly. However when running the code below, I get an api connection error detail: not found.

from litellm import completion
response = completion(
    model="ollama/mistral",
    messages =[ { "content": "What model are you?", "role": "user"}],
    api_base="http://localhost:8000",
)
print('Received:', response)
```

LliteLLM Console output:
`INFO:     127.0.0.1:50416 - "POST /api/generate HTTP/1.1" 404 Not Found`

**Comment by n400peanuts** [comment]
if it's of any help, I solved this error by replacing "ollama/mistral" with "ollama_chat/mistral"
